src/main/java/duke/Event.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Event class represents a Task with a duration.
+ */
+public class Event extends Task {
+    /** Date and time the event starts **/
+    private LocalDateTime startDateTime;
+    /** Date and time the event starts **/
+    private LocalDateTime endDateTime;
+
+    /**
+     * Instantiates an instance of Event.
+     *
+     * @param description Description of the Event task.
+     * @param startDateTime Date and time the event starts.
+     * @param endDateTime Date and time the event ends.
+     */
+    public Event(String description, LocalDateTime startDateTime, LocalDateTime endDateTime) {
+        super(description);
+        this.startDateTime = startDateTime;
+        this.endDateTime = endDateTime;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from: " + this.startDateTime.format(DateTimeFormatter.ofPattern("MMM d yyyy HHmm"))
+                + " to: " + this.endDateTime.format(DateTimeFormatter.ofPattern("MMM d yyyy HHmm")) + ")";
```

Comment:
perhaps can make a separate DatetimeParser class?

 there's many repeated usages of
`.format(DateTimeFormatter.ofPattern("MMM d yyyy HHmm")` in the Event and Deadline classes.